updates/control.sh

```diff
@@ -234,6 +234,20 @@ run_hooks() {
     done
 }
 
+renew_dhcp_after_hwinstalling () {
+    # The Admin IP address of the node is allocated in the
+    # "hardware-installing" transition. Force a renewal here get the new IP
+    # address and avoid it changing and an unexpected time during the
+    # subsequent transitions
+    if [[ $1 = 'hardware-installing' ]]; then
```

Comment:
The ```=``` operator is equivalent to ```==``` here and AFAICS we're using ```=``` everywhere else in this script.